### DISCUSSION TITLE
Vote to retire @fg-j, @ekcasey, and @dumez-k

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#90] [Builder Team]
Hi @paketo-buildpacks/builders-maintainers I'd like to proposing moving @fg-j , @ekcasey , and @dumez-k to the emeritus status.

They have all had valuable contributions, though they have all been inactive for various amounts of time. @fg-j explicitly requested to be moved to emeritus in an offline conversation. This process will remove them from the Builders maintainer (and contributor) teams, and add them under "emeritus" status on: https://github.com/paketo-buildpacks/community

If we prefer to handle each maintainer in a separate thread I'm happy to split the conversation out, too.

Can I get a 👍 / 👎 on moving these three to the emeritus status please?

Thanks,
Rob

**Comment by ryanmoran** [comment]
Done.
